msal4j-sdk/pom.xml

```diff
@@ -181,7 +164,7 @@
     </pluginRepositories>
 
     <build>
-        <sourceDirectory>${project.build.directory}/delombok</sourceDirectory>
+        <sourceDirectory>src/main/java</sourceDirectory>
```

Comment:
nit: leaving this is probably fine but I would expect that to be the default value.